doc/releases/release-notes-3.7.rst

```diff
@@ -1292,6 +1296,13 @@ Networking
 USB
 ***
 
+* new USB device stack:
+
+  * Added support for HID devices
```

Comment:
UAC2 is also brand new class implemented in Zephyr 3.7.0

Reply:
https://docs.zephyrproject.org/latest/releases/release-notes-3.6.html#usb

Reply:
I just double checked and I somehow messed it up yesterday. c83ad53442723a7412e8ae060f5d8bdf142f0274 was indeed before 3.6.0 and 5e3a73de04b99cc25deeb64570bbfc95b0a654df was after 3.6.0.